### PR TITLE
Ingest examples: update application interface envvars

### PR DESCRIPTION
**Pull request by s373r**
⚠️ Please do not merge right after approve!

This PR has strict merge order (see more https://github.com/kamu-data/kamu-cli/issues/302)

### CODE REVIEW

File: content/cli/ingest/ingest-examples.md [hunk 1]
```diff
@@ -225,8 +225,11 @@ The specified container image is expected to conform to the following interface:
 - Use following environment variables:
   - `ODF_LAST_MODIFIED` - last modified time of data from the previous ingest run, if any (in RFC3339 format)
   - `ODF_ETAG` - caching tag of data from the previous ingest run, if any
+  - `ODF_BATCH_SIZE` - batch size for the current ingest run; default value is 10 000, can be overrided via `env`
```
Comment (zaychenko-sergei) [inline content/cli/ingest/ingest-examples.md]: overriden

Comment (sergiimk) [reply content/cli/ingest/ingest-examples.md]: overridden!

Comment (sergiimk) [inline content/cli/ingest/ingest-examples.md]: I think we should provide a more expanded explanation. Tell that this is the recommended number of records, that it's only for scripts that provide continuous stream of data and can resume from previous state.

Comment (s373r) [reply content/cli/ingest/ingest-examples.md]: @sergiimk , yes, it's a good point to highlight the recommendation aspect -- updated

Comment (s373r) [reply content/cli/ingest/ingest-examples.md]: @zaychenko-sergei @sergiimk , fixed

**Review by sergiimk — APPROVED** [review]

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**